mache-chroniclemap/src/main/java/com/excelian/mache/chroniclemap/solr/ArrayIntroSorter.java

```diff
@@ -0,0 +1,58 @@
+package com.excelian.mache.chroniclemap.solr;/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
@tellisnz why is the Solr stuff not included as a binary dependency?...did it need to be altered to fit in with mache?...Or is it a massive jar?...what happens if Solr changes or there's bug fixes?....Does it even matter?